cashier/src/test/java/com/getkeepsafe/cashier/CashierTest.java

```diff
@@ -217,4 +218,22 @@ public void purchase() throws JSONException {
     verify(activity, times(1)).startActivity(new Intent(activity, ShadowActivity.class));
     verifyZeroInteractions(listener);
   }
+
+  @Test
+  public void purchaseDouble() throws  JSONException {
```

Comment:
Can you verify that this test case fails without the changes made above?

Reply:
I am not sure what you are referring to when you say "changes made above." Please be more specific.

Reply:
I ran the test on master and verified that this test case fails.